addons/account/models/partner.py

```diff
@@ -636,7 +636,7 @@ def action_view_partner_invoices(self):
             ('move_type', 'in', ('out_invoice', 'out_refund')),
             ('partner_id', 'in', all_child.ids)
         ]
-        action['context'] = {'default_move_type': 'out_invoice', 'move_type': 'out_invoice', 'journal_type': 'sale', 'search_default_unpaid': 1}
+        action['context'] = {'default_move_type': 'out_invoice', 'move_type': 'out_invoice', 'journal_type': 'sale', 'search_default_unpaid': 1, 'search_default_partner_id': all_child.ids}
```

Comment:
The entire context could be structured alike `action['domain']` / action_vals dictionary in `action_view_partner_with_same_bank` .